From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 18 Aug 2016 19:21:34 +0200
Subject: [PATCH] build: ubuntu mirror: blacklist some SE mirrors

Some SE mirrors are acting up lately, blacklist them:
- ftp.acc.umu.se
- ftp.lysator.liu.se
- ftp.availo.se

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/f_isoroot/f_repobuild/select_ubuntu_repo.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/f_isoroot/f_repobuild/select_ubuntu_repo.sh b/build/f_isoroot/f_repobuild/select_ubuntu_repo.sh
index 4bca6bd..67219cb 100755
--- a/build/f_isoroot/f_repobuild/select_ubuntu_repo.sh
+++ b/build/f_isoroot/f_repobuild/select_ubuntu_repo.sh
@@ -21,7 +21,7 @@ RSYNC="rsync -4 --contimeout 5 --no-motd --list-only"
 # Some Ubuntu mirrors seem less reliable for this type of mirroring -
 # as they are discoved they can be added to the blacklist below in order
 # for them not to be considered.
-BLACKLIST="mirrors.se.eu.kernel.org mirror.its.dal.ca"
+BLACKLIST="mirrors.se.eu.kernel.org mirror.its.dal.ca ftp.acc.umu.se ftp.lysator.liu.se ftp.availo.se"
 
 return_url=0
 
